File: JULY-2025/19-07-2025/p1233.dart
class Solution {
  List<String> removeSubfolders(List<String> folder) {
    folder.sort();
    List<String> ans = [folder.first];

    for (int i = 1; i < folder.length; i++) {
      if (!folder[i].startsWith('${ans.last}/')) {
        ans.add(folder[i]);
      }
    }
    return ans;
  }
}

File: JULY-2025/19-07-2025/p1233.js
/**
 * @param {string[]} folder
 * @return {string[]}
 */
var removeSubfolders = function(folder) {
    folder.sort();
    let filtered = [];

    for (const x of folder) {
        let y = filtered[filtered.length - 1]
        if (filtered.length === 0 || !x.startsWith(y + '/')) {
            filtered.push(x);
        }
    }
    
    return filtered;
};